skbuild/setuptools_wrap.py

```diff
@@ -609,7 +609,13 @@ def setup(*args, **kw):  # noqa: C901
 
     package_prefixes = _collect_package_prefixes(package_dir, packages)
 
-    _classify_installed_files(cmkr.install(), package_data, package_prefixes,
+    # This hook enables custom processing of the cmake manifest
+    cmake_manifest = cmkr.install()
+    process_manifest = kw.get('process_manifest')
+    if process_manifest is not None:
```

Comment:
May be worth checking that `process_manifest` is callable

Reply:
Done, now checking that `process_manifest` is callable.